source/client/app/Minecraft.cpp

```diff
@@ -1163,6 +1163,8 @@ int Minecraft::getFpsIntlCounter()
 
 void Minecraft::leaveGame(bool bCopyMap)
 {
+	Keyboard::setBlocked(true);
```

Comment:
When is the keyboard unblocked?

Reply:
Keyboard::reset() unblocks it - So, no worries...